### ISSUE TITLE
Stop using "kt" as the canonical import for keras_tuner in guides

### ISSUE THREAD

**Issue by mattdangerw** [keras-team/keras-io#862]
Both keras_cv, and keras_nlp will not recommend an abbreviated import. Both for consistency, and clear branding for keras, we should consider dropping the `import keras_tuner as kt` pattern for keras_tuner.

**Comment by mattdangerw** [comment]
See this issue on keras_nlp, for a little more discussion https://github.com/keras-team/keras-nlp/issues/134

@haifeng-jin was curious your thoughts here. Checked with @fchollet who thought it would probably make sense to encourage imports with the name "keras".

**Comment by haifeng-jin** [comment]
Agreed. It is also easier for the user to remember `keras_tuner` than `kt` actually.
I will create a PR.
